apps/students/models.py

```diff
@@ -48,6 +55,11 @@ def get_profile_image_url(self):
         else:
             return settings.STATIC_URL + 'home/images/man.png'
 
+    @property
+    def profile_url(self):
+        """Returns the url to volunteer profile."""
```

Comment:
please change the comment to 'student profile'

Reply:
Done